common/chat/helper.ts

```diff
@@ -24,4 +27,60 @@ const createChatSignature = async (user: User): Promise<string> => {
     return hash.digest('hex');
 };
 
-export { userIdToTalkJsId, parseUnderscoreToSlash, checkResponseStatus, createChatSignature };
+const getUserIdsForChatParticipants = (participantIds: string[]): [studentId: number, pupilId: number] => {
```

Comment:
What is the purpose of this function, returning the last pupil and student in the participants?